### PR TITLE
Make xterm test more robust

### PR DESCRIPTION
**Pull request by kravciak**
Fixing poo#13708
Local run:
SLES: http://dhcp91.suse.cz/tests/2475 (gnome)
TW: http://dhcp91.suse.cz/tests/2476 (kde)

### CODE REVIEW

File: tests/x11/xterm.pm [hunk 1]
```diff
@@ -11,20 +11,14 @@
...
 use strict;
 use testapi;
-use utils;
 
 sub run() {
-    my $self = shift;
     mouse_hide(1);
     x11_start_program("xterm");
-    sleep 2;
-    type_string "cd\n";
```
Comment (kravciak) [inline tests/x11/xterm.pm]: @okurz why we need to "cd" home on KDE - without softfail?

File: tests/x11/xterm.pm [hunk 2]
```diff
@@ -11,20 +11,14 @@
...
-    sleep 1;    # go to $HOME (for KDE)
-    clear_console;
+    assert_screen('xterm-started');
     for (1 .. 13) { send_key "ret" }
     type_string "echo If you can see this text xterm is working.\n";
-    sleep 2;
-    assert_screen 'test-xterm-1', 3;
+    assert_screen 'test-xterm-1', 10;
```
Comment (dzedro) [inline tests/x11/xterm.pm]: leave default ?

Comment (kravciak) [inline tests/x11/xterm.pm]: ok ok, default used..

File: tests/x11/xterm.pm (hunk 1 above)
Comment (okurz) [inline tests/x11/xterm.pm]: I don't know. I didn't do this. But it helps to keep the prompt short so I would recommend to keep it in all cases … and delete the comment stating the obvious and also in a wrong way

**Comment by okurz** [comment]
you changed my mind ;-)

**Pull request merged**